tests/test_cloudfront.py

```diff
@@ -217,6 +217,33 @@ def test_distribution_check_s3_origin_missing_bucket(self):
         self.assertEqual(len(resources), 1)
         self.assertEqual(resources[0]['c7n:mismatched-s3-origin'][0], 'c7n-idontexist')
 
+    def test_distribution_check_logging_enabled(self):
+        factory = self.replay_flight_data("test_distribution_check_logging_enabled")
+
+        p = self.load_policy(
+            {
+                "name": "test_distribution_logging_enabled",
+                "resource": "distribution",
+                "filters": [
+                    {
+                        "type": "distribution-config",
+                        "key": "Logging.Enabled",
+                        "value": True
+                    }
+                ]
+            },
+            session_factory=factory,
+        )
+
+        resources = p.run()
+
+        self.assertEqual(len(resources), 1)
+
+        client = local_session(factory).client("cloudfront")
+        dist_id = resources[0]['Id']
+        resp = client.get_distribution_config(Id=dist_id)
```

Comment:
we don't need to refetch the resource if we're not modifying it, just check the resource attributes and annotations from the filter.